libz-rs-sys/src/gz.rs

```diff
@@ -250,7 +250,10 @@ unsafe fn gzopen_help(source: Source, mode: *const c_char) -> gzFile {
             #[cfg(target_os = "linux")]
             {
                 oflag |= libc::O_LARGEFILE;
-                if cloexec {
+            }
+            if cloexec {
+                #[cfg(target_os = "linux")]
+                {
```

Comment:
are you working around an unused error on `cloexec` here? if so, you can rename the variable to `_cloexec`, which will  not trigger that error, but still allow you to use the value in the linux-specific code.

Reply:
Yes, it's for non-Linux targets (I ran into it on macOS). I used this approach instead of the leading underscore workaround (`_cloexec`) so that on Linux the compiler can still verify that `cloexec` is being used.

Reply:
allright, that's fine then.